doc/source/ray-overview/examples.rst

```diff
@@ -619,22 +619,10 @@ Ray Examples
 
     .. grid-item-card:: :bdg-secondary:`Code example`
         :class-item: gallery-item reinforcement-learning rllib
-        :link: https://github.com/ray-project/ray/blob/master/rllib/examples/batch_norm_model.py
+        :link: https://github.com/ray-project/ray/blob/master/rllib/examples/models/batch_norm_model.py
```

Comment:
these are two different files. the original one (already deleted) has a `__main__` and is directly runnable, where this one is not.

same with many other ones.

Reply:
I am not really the owner

@sven1977 ?